R/tcplCytoPt.R

```diff
@@ -149,27 +150,31 @@ tcplCytoPt <- function(chid = NULL, aeid = NULL, flag = TRUE,
   } else {
     mtst <- 0
   }
-  print(5)
+  cat("5: Loading level 5 data\n")
```

Comment:
I wonder if here we need to use "add.fld" to tcplLoadData() to make sure we get the mc5 parameters? In my error message, "modl_ga" column is not available so I'm wondering where it should come into play.